packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -43,7 +44,8 @@ async function processDeadLetterArchive({
     const deadLetterObjects = listObjectsResponse.Contents;
     const promises = await Promise.allSettled(deadLetterObjects.map(
       async (deadLetterObject) => {
-        const cumulusMessage = await getJsonS3Object(bucket, deadLetterObject.Key);
+        const deadLetterMessage = await getJsonS3Object(bucket, deadLetterObject.Key);
+        const cumulusMessage = await unwrapDeadLetterCumulusMessage(deadLetterMessage);
```

Comment:
This is the crux of the first part of the change.   It's possible for messages to either be written as a SQS message, *or* a Cumulus message depending on source.    

Options are to either parse both options or standardize output, however existing messages are *already* both.